src/odemis/driver/symphotime.py

```diff
@@ -690,12 +720,36 @@ def StartMeasurement(self):
 
     def StopMeasurement(self):
         '''
-        Sends a user break to the server to stop the measurement. 
+        Sends a user break to the server to stop the measurement. Note that the measurement state will
+        not change until the server acknowledges.
         '''
         if self.isMeasuring():
             logging.info("Sending user break.")
             self._sendMessage(EncodedStatusMessage (PQ_STOPREASON_CODE_USER_BREAK))
             self._user_break = True
+            
+    def CheckMeasurement(self, meas_type):
```

Comment:
IMHO, it's a little over-the-top to create a whole function just to do 2 ifs, and called from only one site, which has only two lines. I'd have just merged the two functions together.